src/components/Spoiler.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <b-collapse :id="`collapsible-${this.cid}`" @hidden="reflow()" @shown="reflow()">
```

Comment:
margin-bottom:  of 3-4px; should fix the  button border problem

Reply:
I thought of the same thing. I'll just add something similar for the Collapsible component.

Reply:
also I think you meant 'margin-top'

Reply:
It should be fixed now.